modules/azure_functions/devdoc/azure_functions.md

```diff
@@ -161,26 +161,26 @@ This function will be the main work of this module. The processing of each
 message in pseudocode is as follows:
 
 
-01: Retrieves the content of the message received;
+01: Retrieve the content of the message received
 
-02: Create an HTTPAPIEX handle;
+02: Create an HTTPAPIEX handle
 
 03: Clone the relative path handle
 
-04: Adds to the  `name` parameter together with an ID (Hard Coded) to the relative path;
+04: Add to the `name` parameter together with an ID (Hard Coded) to the relative path
 
-05: adds the content of the message as a body of the HTTP POST, content type application/json;
+05: Add the content of the message as a body of the HTTP POST, content type application/json
 
-06: Adds securityKey as HTTP Header, if security key string exists;
+06: Add securityKey in HTTP Header, if security key string exists
 
-07: call HTTPAPIEX_ExecuteRequest to send the message;
+07: Call HTTPAPIEX_ExecuteRequest to send the message
 
-08: logs the reply back by the Azure Functions.
+08: Log the reply back by the Azure Functions
 
 
-**SRS_AZUREFUNCTIONS_04_010: [** If `moduleHandle` is NULL than `AzureFunctions_Receive` shall fail and return. **]**
+**SRS_AZUREFUNCTIONS_04_010: [** If `moduleHandle` is NULL then `AzureFunctions_Receive` shall fail and return. **]**
```

Comment:
Thanks for this change. 
This needs to be changed on the code as well (Since it's a requirement tag). 
https://github.com/Azure/azure-iot-gateway-sdk/blob/master/modules/azure_functions/src/azure_functions.c
Lines: 294 and 295.

Thanks!